src/callback_requests.rs

```diff
@@ -209,7 +204,11 @@ pub enum CallbackRequestType {
     },
 
     KubernetesCanI {
-        subject_access_review: Box<SubjectAccessReviewWrapper>,
+        user: String,
+        group: String,
+        namespace: String,
+        resource: String,
+        verb: String,
         disable_cache: bool,
```

Comment:
This will have to be updated to the new type I proposed inside of the SDK. You should be able to embed the `ResourceAttribute` straight into this enum.

BTW, please add inline docs

Reply:
I've added the `SubjectAccessReviewRequest` directly on the enum. Therefore, I convert it into `SubjectAccessReview` when necessary later in the code. Before calling Kubernetes API. Take a look on it (and in the sdk trait implementations) and let me know what do you think.

Reply:
Thanks, now one could test for Groups, I consider this resolved.